rpm/droid-hal-kirin.spec

```diff
@@ -1,4 +1,5 @@
 %define device kirin
+%define rpm_device kirin
 
```

Comment:
I dislike the confusion this creates for anyone reading this code, and also repetitiveness below.
There are other ways to fix, such as adding `Source0: droid-hal-%{device}-%{version}.tar.bz2`, but would %{version} get resolved at this point, needs test first

Reply:
I am not sure if adding "Source0: droid-hal-%{device}-%{version}.tar.bz2" would help because https://github.com/mer-hybris/droid-hal-device/blob/master/droid-hal-device.inc#L141 will overwrite it.

Reply:
I already had our Friday's IRC conversation in mind, where we'd have to remove Source0 from the submodule.
To quickly test if normal build doesn't break, I ran a version on OBS with a Source1000 line in this .spec

Reply:
So how about removing this line https://github.com/mer-hybris/droid-hal-device/blob/bfee205f3e58da705fda296ca6387be2b8cdcb2d/droid-hal-device.inc#L141 and adding `Source0: droid-hal-%{device}-%{version}.tar.bz2` to .spec here to aid promotions.

Reply:
Ok, let's go with that way, I'll prepare changes.